src/Commands/BuildCommand.php

```diff
@@ -34,6 +34,19 @@ public function handle(): void
                 echo $output;
             });
 
+        // Install electron plugin as a tarball (avoid bundling symlinks)
+        Process::path(__DIR__.'/../../resources/js/electron-plugin')
+            ->run('npm pack', function (string $type, string $output) {
+                echo $output;
+            });
+
+        Process::path(__DIR__.'/../../resources/js')
+            ->env($this->getEnvironmentVariables())
+            ->forever()
+            ->run('npm ci ./electron-plugin/nativephp-electron-plugin-tarball.tgz', function (string $type, string $output) {
```

Comment:
We need to do a clean install here, because the tarball version was locked. It won't update when changes where made to the electron-plugin otherwise